### DISCUSSION TITLE
Login user and password for http://IP:4000 don't work.

### DISCUSSION THREAD

**Discussion by hofmarkus** [tillsteinbach/VWsFriend#187] [Q&A] [answered]
Since the new version of VWsFriend I see on the right side:
Warnings:
. Some panels need the battery capacity of Eup to be set. Please set the size here: Vehicle Parameters and then I should change the parameters at http://IP:4000.

I have a question about the login at http://IP:4000. I always get a login request. In the field User I enter the data from DB_USER (from my .env file) and Passwod DB_Password. I have also tried the data with VWSFRIEND_USERNAME and VWSFRIEND_PASSWORD but unfortunately without success. 
On http://IP:3000 the LOGIN with VWSFRIEND_USERNAME and VWSFRIEND_PASSWORD works without problems.
How can I possibly reset the user at PORT 4000 or assign a new password?
Thanks for your help in advance.

**Comment by tillsteinbach** [comment, marked as answer]
You need to update the docker-compose.yml file once after this update. Afterwards the same password like in Grafana will work again.

**Reply by hofmarkus** [reply]
Unfortunately, I have after the last update again the same problem with LOGIN to the web interface http://IP-ADDRESS:4000. It is always displayed user and password here. I have also updated the file docker-compose.yml like last time, but this time without success. What i could do to resolve the problem?

**Reply by tillsteinbach** [reply]
What do you mean with "always"? Is it not accepting your user/password or do you have to type it in from time to time?

**Reply by hofmarkus** [reply]
This is my mistake. So after the update I was again prompted to enter the user and password at http://IP-ADDRESS:4000. Thought that one should update like last time docker-compose.yml. Now after entering user + password everything works normal again. Thanks anyway and sorry for false alarm.

**Reply by tillsteinbach** [reply]
No problem, good to hear it works!

**Comment by hofmarkus** [comment]
Thank you, that was the solution.
